### PR TITLE
feat: add new post on roadmap updates

### PR DESCRIPTION
**Pull request by samajammin**
# Description

Add new blog post with MACI roadmap updates

## Related issue(s)

N/A

## Confirmation

- [x] I have read and understand MACI's [contributor guidelines](https://github.com/privacy-scaling-explorations/maci/blob/dev/CONTRIBUTING.md) and [code of conduct](https://github.com/privacy-scaling-explorations/maci/blob/dev/CODE_OF_CONDUCT.md).
- [x] I have read and understand MACI's [GitHub processes](https://github.com/privacy-scaling-explorations/maci/discussions/847).
- [x] I have read and understand MACI's [testing guide](https://maci.pse.dev/docs/testing).

### CODE REVIEW

File: website/blog/2024-01-10-roadmap.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+## Project updates
+
+New year, new team, new roadmap!
+
+### 1) New team
+
+Over the past few months we've undergone lot of change within the MACI core team. I'd like to take this chance to introduce and welcome our new core members!
+
+MACI is now maintained and will be continuously improved by our core team:
```
Comment (samajammin) [inline website/blog/2024-01-10-roadmap.md:44]: @ctrlc03 @0xmad @kittybest @crisgarner @aguzmant103 Vee - feel free to update (or tell me) which URL (GH, Twitter, personal website, etc.) you'd like you use here

File: website/blog/2024-01-10-roadmap.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+## Project updates
+
+New year, new team, new roadmap!
+
+### 1) New team
+
+Over the past few months we've undergone a lot of changes within MACI's core team. I'd like to take this chance to introduce and welcome our new core members!
+
+MACI is now maintained and will be continuously be improved by our core team, which consists of:
```
Comment (samajammin) [inline website/blog/2024-01-10-roadmap.md:44]: @ctrlc03 @0xmad @kittybest @crisgarner @aguzmant103 @Vee-18 - feel free to update (or tell me) which name & URL (GH, Twitter, personal website, etc.) you'd like you use here

File: website/blog/2024-01-10-roadmap.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+### 2) New roadmap
+
+As a team that's building in the open source space, we're taking a concerted effort to make our work more accessible and to foster more collaboration and feedback from our community. As part of this work, we're making our core team's roadmap public and will continue to do so.
+
+[Read our full 2024 roadmap here](/roadmap).
+
+TL;DR: We'll be focusing most of our time on:
+
+1. **Developer experience** (documentation, tooling, refactoring) so that MACI is easier for projects to integrate
+2. **Community engagement** (integration support, hackathon bounties, conference presentations) to more quickly iterate and improve based off of feedback
```
Comment (kittybest) [inline website/blog/2024-01-10-roadmap.md:72]: maybe typo here? based off feedback

**Review by aguzmant103 — APPROVED** [review]
Lgtm!

**Review by 0xmad — APPROVED** [review]
@samajammin thanks, just one comment about commit styles. Could you use docs prefix instead of feat. In the future it will be important to not trigger version bump in case of docs/blogs/tests changes.

File: website/blog/2024-01-10-roadmap.md (hunk 1 above)
Comment (crisgarner) [reply website/blog/2024-01-10-roadmap.md]: LGTM! I think if everyone uses GitHub, it makes sense to keep all the same

**Review by crisgarner — APPROVED** [review]

**Review by kittybest — APPROVED** [review]

**Comment by samajammin** [comment]
> @samajammin thanks, just one comment about commit styles. Could you use docs prefix instead of feat. In the future it will be important to not trigger version bump in case of docs/blogs/tests changes.

Great call! Noted

**Pull request merged**